internal/server/metrics.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func setupMetrics(db *gorm.DB) *prometheus.Registry {
-	registry := metrics.NewRegistry()
+	registry := metrics.NewRegistry(apiVersion())
```

Comment:
This should be `internal.FullVersion()` as well

Reply:
It can't reference `internal.FullVersion` directly. We need the variable to patch it.

But I can change the variable from `apiVersion`, to `productVersion`. Would that work?

Reply:
I pushed another commit which renames the variable and the patch function to `productVersion`